index.js

```diff
@@ -7,8 +7,8 @@
 
 'use strict';
 
-const lower = 'abcdefghijklmnopqrstuvwxyz';
-const upper = lower.toUpperCase();
+var lower = 'abcdefghijklmnopqrstuvwxyz';
```

Comment:
@Rudxain,
Why changing const to var ?

Reply:
Because it's a "breaking change" in semver. Using `var` only requires a "patch" increment, but `const` is only compatible with non-legacy versions of Node, therefore it requires a "major" increment.

However, TBF, most repos don't consider newer language features to be "breaking", because the public API stays identical.

But even if `version` (in the package.json) isn't major-incremented, the following must happen anyways:
```diff
 "engines": {
-   "node": ">=0.10.0"
+   "node": ">=6.0.0"
 }
```

Reply:
@Rudxain,
I see, thanks for your quick reply 🙂